src/overlays/actors/ovl_Obj_Tokei_Tobira/z_obj_tokei_tobira.c

```diff
@@ -36,16 +37,150 @@ static InitChainEntry D_80ABD750[] = {
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
+Vec3f D_80ABD760 = { 0.0f, 0.0f, 80.0f };
+s16 D_80ABD76C[2] = { 0xC000, 0x4000 };
+s32 D_80ABD770[2] = { 0x060012B0, 0x06001590 };
+f32 D_80ABD778[2] = { 1.0f, -1.0f };
+Gfx *D_80ABD780[4] = { (Gfx *)0x06001108, (Gfx *)0x060013E8, NULL, NULL };
```

Comment:
By looking at how `D_80ABD780` is used, you can see it only cares about the first 2 values of this array. The other two values are just automatically generated padding (because data sections are padded to a 0x10 boundary)
In other words, you can remove the last two `NULL`s
```suggestion
Gfx *D_80ABD780[] = { (Gfx *)0x06001108, (Gfx *)0x060013E8, };
```
Also, as you can see this array is actually an array of pointers but instead of having symbols it has raw numbers, this needs to be fixed. The tutorial covers how to do it here: https://github.com/zeldaret/mm/blob/master/docs/tutorial/data.md#segmented-pointers-and-object-symbols